lib/rules/length-zero-no-unit/index.js

```diff
@@ -40,6 +40,15 @@ function rule(actual, secondary, context) {
 			const ignorableIndexes = new Array(stringValue.length).fill(false);
 			const parsedValue = valueParser(stringValue);
 
+			if (decl.prop.toLowerCase() === 'flex') {
```

Comment:
May just want to add a comment pointing to the issue / MDN docs regarding the flex ambiguity - but will defer to others if they feel like this is appropriate!

Reply:
@jeddy3 Should I add an inline comment here or issue number in commit message is enough?

Reply:
An inline comment would be nice as this is tricky stuff. Perhaps:

```
// Allow <width> for last value in two-value and three-value flex syntax
// https://developer.mozilla.org/en-US/docs/Web/CSS/flex#syntax
```